### PR TITLE
GDB-9776: Add random password and cluster secret

### PR DESCRIPTION
**Pull request by kristianiliev1**
## Description
Added random password and cluster token generate for GraphDB Admin password.

Removed some validations that were not needed here.

## Related Issues
https://ontotext.atlassian.net/browse/GDB-9776

## Checklist

- [X] I have tested these changes thoroughly.
- [X] My code follows the project's coding style.
- [X] I have added appropriate comments to my code, especially in complex areas.
- [X] All new and existing tests passed locally.

### CODE REVIEW

File: modules/config/variables.tf [hunk 1]
```diff
@@ -15,18 +15,19 @@ variable "resource_name_prefix" {
...
-  default     = "s3cret"
+  default     = null
+  sensitive   = true
 
-  validation {
-    condition     = length(var.graphdb_admin_password) >= 5 && can(regex("[0-9]", var.graphdb_admin_password))
-    error_message = "Password must be at least 5 characters long and contain at least one number."
-  }
+  # validation {
+  #   condition     = length(var.graphdb_admin_password) >= 5 && can(regex("[0-9]", var.graphdb_admin_password))
```
Comment (viktor-ribchev) [inline modules/config/variables.tf:22]: Why?

File: modules/config/variables.tf [hunk 2]
```diff
@@ -15,18 +15,19 @@ variable "resource_name_prefix" {
 variable "graphdb_admin_password" {
   description = "Password for the 'admin' user in GraphDB."
   type        = string
-  default     = "s3cret"
+  default     = null
```
Comment (viktor-ribchev) [inline modules/config/variables.tf:18]: default should be in the root variables.tf

File: modules/config/variables.tf [hunk 3]
```diff
@@ -15,18 +15,19 @@ variable "resource_name_prefix" {
...
+  #   condition     = length(var.graphdb_admin_password) >= 5 && can(regex("[0-9]", var.graphdb_admin_password))
+  #   error_message = "Password must be at least 5 characters long and contain at least one number."
+  # }
 }
 
 variable "graphdb_cluster_token" {
   description = "Cluster token used for authenticating the communication between the nodes."
-  type        = string
-  default     = "s3cret"
+  default     = null
```
Comment (viktor-ribchev) [inline modules/config/variables.tf:29]: default should be in the root variables.tf

File: modules/config/main.tf [hunk 4]
```diff
@@ -2,14 +2,14 @@ resource "aws_ssm_parameter" "graphdb_admin_password" {
   name        = "/${var.resource_name_prefix}/graphdb/admin_password"
   description = "Password for the 'admin' user in GraphDB."
   type        = "SecureString"
-  value       = var.graphdb_admin_password
+  value       = base64encode(local.graphdb_admin_password)
```
Comment (viktor-ribchev) [inline modules/config/main.tf:5]: You should also update the user data scripts

File: modules/config/main.tf [hunk 5]
```diff
@@ -2,14 +2,14 @@ resource "aws_ssm_parameter" "graphdb_admin_password" {
...
 }
 
 resource "aws_ssm_parameter" "graphdb_cluster_token" {
   name        = "/${var.resource_name_prefix}/graphdb/cluster_token"
   description = "Cluster token used for authenticating the communication between the nodes."
   type        = "SecureString"
-  value       = var.graphdb_cluster_token
+  value       = base64encode(local.graphdb_cluster_token)
```
Comment (viktor-ribchev) [inline modules/config/main.tf:12]: You should also update the user data scripts

File: modules/config/variables.tf (hunk 1 above)
Comment (viktor-ribchev) [reply modules/config/variables.tf:22]: And what if the password is provided by the user? Either we should change the condition or remove it entirely.

**Review by yaskoo — APPROVED** [review]

**Pull request merged**